index.html

```diff
@@ -699,6 +699,70 @@ <h2>
     const observer = new ComputePressureObserver(pressureChange);
     observer.observe("cpu");
   </pre>
+  <p>
+    In the following example, we want to demonstrate the usage of {{ComputePressureObserver/takeRecords()}},
+    by retrieving the remaining |entries| accumulated since the last callback.
+  </p>
+  <p>
+    It is recommended to do so before {{ComputePressureObserver/disconnect()}}, otherwise {{ComputePressureObserver/disconnect()}} will clear them and they will be lost.
+  </p>
+  <pre class="example js" title="Usage of takeRecords">
+    function pressureChange(entries) {
+      // do something with entries
+    }
+
+    const observer = new ComputePressureObserver(pressureChange);
+    observer.observe("cpu");
+
+    // observe.disconnect() will be called promptly, no time to wait for
+    // callback. latest entries are needed immediately or will be lost otherwise.
+    const entries = takeRecords();
+
+    // do something with entries.
+
+    observer.disconnect();
+  </pre>
+  <p>
+    In the following example, we want to demonstrate the usage of {{ComputePressureObserver/unobserve()}}.
+    {{ComputePressureObserver/unobserve()}} will stop observing the specified |target:ComputePressureTarget|.
+    It will not clear the pending |entries| collected since the last callback.
+  </p>
+  <pre class="example js" title="Usage of unobserve">
+    function pressureChange(entries) {
+      // do something with entries
+    }
+
+    const observer = new ComputePressureObserver(pressureChange);
+    observer.observe("cpu");
+
+    // some time later...
+
+    observer.unobserve("cpu");
+
+    // expected to have entries if callback pressureChange hasn't been called.
```

Comment:
FWIW I naively thought entries would be empty at that point.

Reply:
I wonder what is the right approach here, it seems that IntersectionObserver doesn't even clear [[QueuedEntries]] on disconnect (that looks like a bug)

Reply:
https://w3c.github.io/IntersectionObserver/#dom-intersectionobserver-disconnect

Reply:
MutationObserver definitely empties it (but doesn't have unobserve):

https://dom.spec.whatwg.org/#dom-mutationobserver-disconnect